net-wireless/uhd/files/uhd-4.0.0.0-fix-frequency-offsets.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/host/lib/usrp/usrp2/usrp2_impl.cpp b/host/lib/usrp/usrp2/usrp2_impl.cpp
```

Comment:
Where is this from? Try to [include](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto) information on where patches are from and other background

Reply:
I add some information in these patch file, 
uhd-4.0.0.0-fix-frequency-offsets.patch from https://patch-diff.githubusercontent.com/raw/EttusResearch/uhd/pull/439.diff.
uhd-4.0.0.0-fix-boost-math-header.patch and uhd-4.0.0.0-fix-gcc11-sleep-for.patch write by me.